docker-compose.yml

```diff
@@ -5,14 +5,25 @@ version: "3.9"
 services:
   auto-gpt:
     depends_on:
-      - redis
+      - redisStack
     build: ./
     env_file:
       - .env
     volumes:
-      - "./autogpt:/home/appuser"
-      - ".env:/home/appuser/.env"
+      - "./autogpt:/app"
+      - ".env:/app/.env"
+      - "$PWD/auto_gpt_workspace:/home/appuser/auto_gpt_workspace"
+      - "./ai_settings.yaml:/home/appuser/ai_settings.yaml"
     profiles: ["exclude-from-up"]
 
-  redis:
-    image: "redis/redis-stack-server:latest"
+  redisStack:
```

Comment:
There is a reference to redis in the .env.template that would be worth updating
```
## REDIS_HOST - Redis host (Default: localhost, use "redis" for docker-compose)
```

Reply:
thanks. updated.